### ISSUE TITLE
FR: find orphans (notes with no connections)

### ISSUE THREAD

**Issue by csss1234** [SkepticMystic/breadcrumbs#576]
i use breadcrumbs because without hierarchy, I worry notes are getting “lost”. I might take a note on the same thing three times, because I titled it differently each time.

since discovering BC, im going through my vault looking for disconnected notes. Basically I want every note to have an “up”. 

I developed a workaround to reveal these orphan notes, but it takes a few steps and needs to be manually refreshed 

1. Go to top/master index note and use BC to “create index”
2. Paste in note along with dataview to show all notes *not* linked to by this index:
```dataview
List

FROM "" AND -"path/to/exclude" AND -#tag_to_exclude

AND -outgoing([[Note where I pasted index]]) 

SORT file.mtime asc
Limit 40
```
Or take it a step further by also excluding any notes one-link-removed (notes linked to notes linked to Index) by including:

`WHERE !filter([[Note where I pasted index]].file.outlinks, (o) => contains(file.outlinks, o) OR contains(file.inlinks, o))`

**problem: needs manual refresh**

After finding and connecting some orphans, you have to go back to the master/index note, recreate the index with BC, and paste it again. 

 Obsidians built-in graph view could be used to reveal orphans (again after using BC to create index), but that would involve looking at the graph view 😵‍💫

**Comment by jacobmichels** [comment]
I agree that this would be a cool feature to have built in. Thanks for sharing your workaround!

> Obsidians built-in graph view could be used to reveal orphans (again after using BC to create index), but that would involve looking at the graph view 😵‍💫

lol, I too am overwhelmed by this view. It looks cool I guess, but I'm not sure how to extract value from it

**Comment by michaelpporter** [comment]
<img width="758" height="197" alt="Image" src="https://github.com/user-attachments/assets/c85803e6-c44b-421a-9acd-e1eb9548f548" />

[Version 4](https://github.com/SkepticMystic/breadcrumbs/blob/master/V4.md) has refresh on view change. Have you tried this?

**Comment by csss1234** [comment]
@michaelpporter No, sorry that is not what I meant by "Refresh". Perhaps I should have used the word "Rebuild", as in the index needs to be manually rebuilt every time I want to search for orphans.

**Comment by michaelpporter** [comment]
After some thinking, I think I understand the current flow. I also do not see a clean solution.

1. Creating an index copies to the clipboard allowing the creation of a static note.
2. Then using dataview to find notes not linked to the note, i.e. has no breadcrumbs

It might be better to make a Dataview that finds notes without the breadcrumb fields.

Perhaps, saving the index to a file like the [Find orphaned files](https://github.com/Vinzent03/find-unlinked-files/tree/main) does.
